gamemodes/terrortown/gamemode/shared/sh_shop.lua

```diff
@@ -30,6 +30,15 @@ shop.buyTable = shop.buyTable or {}
 shop.globalBuyTable = shop.globalBuyTable or {}
 shop.teamBuyTable = shop.teamBuyTable or {}
 
+---
+-- Initializes all necessary variables for the shop
+-- @realm shared
+function shop.Initialize()
+    if CLIENT then
+        shop.favorites.orm = orm.Make(shop.favorites.databaseName)
```

Comment:
@Histalek does the orm really need a late init? Sounds like something we should address in the future and should rather not be needed

Reply:
Currently it does and gets all the necessary information out of the sql table. If it does not exist, it can't do that. So it is only needed for the first time migrations are done. 
https://github.com/TTT-2/TTT2/blob/master/lua/ttt2/libraries/orm.lua#L25-L32

Reply:
> @Histalek does the orm really need a late init? Sounds like something we should address in the future and should rather not be needed

The orm does need to be initialized after any migrations, yes. `orm.Make` creates its data based on the current schema of the given database table, if you change the schema afterwards you are in for a bad time.

The explizit `Initialize` here is only necessary to guarantee that migrations have run.

Reply:
I understand but dont like it gotta see if we can redo that later